modules/virt-attaching-vm-to-sriov-network.adoc

```diff
@@ -19,21 +19,24 @@ spec:
   domain:
     devices:
       interfaces:
-      - masquerade: {} <1>
-        name: default
-      - name: <sriov-network> <2>
+      - name: default <1>
+        masquerade: {} <2>
+      - name: nic1 <3>
         sriov: {}
   networks:
-  - name: default <3>
+  - name: default <4>
     pod: {}
-  - multus:
-    name: <sriov-network> <4>
+  - name: nic1 <5>
+    multus:
+        networkName: <sriov-network> <4>
 ...
 ----
-<1> The `masquerade` binding to the default Pod network.
-<2> The name of the SR-IOV network.
-<3> The default Pod network.
-<4> The name of the SR-IOV network. This is the same value as the `interfaces.name`.
+<1> A name for the interface that is connect to the Pod network.
+<2> The `masquerade` binding to the default Pod network.
+<3> A name for the SR-IOV interface.
```

Comment:
To keep the articles consistent:
```suggestion
<3> The name for the SR-IOV interface.
```

Reply:
my use of "a" is intentional. "the name" alludes to a name that already exists somewhere else. But here, it is just a random name defined for this VM only and exists nowhere else.

Reply:
In that case we tend to use "A unique name for .." to clarify that the user is defining the name. If that works for you then please apply to L34 and L36
We should also wrap the names in <names> in the config but I'll raise a separate comment above.